osu.Game/Database/RealmBackedStore.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Platform;
+
+#nullable enable
+
+namespace osu.Game.Database
+{
+    public abstract class RealmBackedStore
+    {
+        protected readonly Storage? Storage;
```

Comment:
what was the intention behind this storage?

Reply:
Copied from `DatabaseBackedStore`; will likely be required in the future. Can be removed until store comes along which requires it, if that's a better direction.